frontend/src/APIClients/queries/UserQueries.ts

```diff
@@ -9,24 +21,28 @@ export type User = {
   approvedLanguagesReview: string;
 } | null;
 
-export const buildUsersQuery = () => {
-  // TODO: Modify when users query is modified
+export const buildUsersQuery = (
+  isTranslators: boolean,
+  language?: string,
+  level?: number,
+  nameOrEmail?: string,
+) => {
+  const queryParams = `
```

Comment:
oof, the linter made quite a mess out of this. Maybe something like this could be more readable? what do you think?
```
let queryParams =  `${language ? `language: "${language}", ` : ""}`  
queryParams += `${level ? `level: ${level}, ` : ""}`
queryParams += `${nameOrEmail ? `nameOrEmail: "${nameOrEmail}", ` : ""}`;
```
alternatively, we can disable the linting in this code block and just straighten out the spacing a little:
```
/* eslint-disable my-error */
 const queryParams = `
 ${language ? `language: "${language}", ` : ""} 
 ${level ? `level: ${level}, ` : ""} 
 ${nameOrEmail ? `nameOrEmail: "${nameOrEmail}", ` : ""}
  `;
/* eslint-disable my-error */
```